src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -674,6 +669,15 @@ public static boolean isClusterPerm(String action0) {
         );
     }
 
+    static boolean isDnfofAction(String action0) {
```

Comment:
I think `WildcardMatcher` can be used here. Whdyt?

Reply:
Used in : [c12333c](https://github.com/opensearch-project/security/pull/3236/commits/c12333ce5e1442bdd206d23664a5b71e43787203)